lib/ui/main_menu.dart

```diff
@@ -32,8 +32,8 @@ CloudVisionApi? vision = CloudVisionApi();
 bool? _completed;
 
 class MainWidgetState extends State<MainWidget> {
-  DateTime selectedDate = DateTime.now();
-  String mailType = "Email";
+  DateTime selectedDate = DateTime.now() /*.subtract(const Duration(days: 1)) */;
```

Comment:
remove comment

Reply:
yeah thats fine, it can be deleted, I just missed that one.  To have it select a specific date for a digest, I made it subtract a day since I didn't have a digest on that day, and it kept giving me an error.  But I think some of this may be irrelevant now that the digest is obtained automatically.  I got the impression from Mike that the Digest button on the main menu isn't even really required anymore.

Reply:
the digest view is still required, it just isn't a part of what you're working now in this PR